### PR TITLE
[onert_run] Revise allocation and session access

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit moves allocation and session access from each formatters and random generator into main function. 
It will reduce duplication and simplify formatter and generator's logic to fill and dump data.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

PR to prepare #13008

### CODE REVIEW

File: tests/tools/onert_run/src/rawformatter.h [hunk 1]
```diff
@@ -28,13 +27,13 @@ struct nnfw_session;
 
 namespace onert_run
 {
-class RawFormatter : public Formatter
+class RawFormatter
```
Comment (ragmani) [inline tests/tools/onert_run/src/rawformatter.h:30]: Is there any possibility that `RawFormatter` is used through `Formatter` interface?

Comment (hseok-oh) [reply tests/tools/onert_run/src/rawformatter.h:30]: I can't find that.

**Comment by hseok-oh** [comment]
@nnfw-bot test onert-android-release

**Review by ragmani — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**